compiler/src/iree/compiler/InputConversion/MHLO/MHLOToLinalgOnTensors.cpp

```diff
@@ -426,6 +428,18 @@ struct ConvertMHLOToLinalgOnTensorsPass
                                       std::move(patterns)))) {
       return signalPassFailure();
     }
+
+    {
+      // Apply the patterns to remove unused operands and results.
+      RewritePatternSet removeUnusedOperandsResultsPatterns(&getContext());
```

Comment:
I'd just rename it to `patterns` because it is shorter; the scope and the comment are good enough to me.

Reply:
I left it as is. There is patterns in a parent scope, so the name clash can be hard to read. This makes it more explicit.